plugin/backend_linux.go

```diff
@@ -53,12 +60,33 @@ func (pm *Manager) Enable(name string, config *types.PluginEnableConfig) error {
 }
 
 // Inspect examines a plugin config
-func (pm *Manager) Inspect(name string) (tp types.Plugin, err error) {
-	p, err := pm.pluginStore.GetByName(name)
-	if err != nil {
+func (pm *Manager) Inspect(refOrID string) (tp types.Plugin, err error) {
+	// Match on full ID
+	pluginID := strings.ToLower(refOrID)
```

Comment:
For containers and images, we expect the lower case IDs as input. Doing this just for plugins, would make an inconsistent user experience.